packages/@webex/plugin-meetings/test/integration/spec/journey.js

```diff
@@ -44,13 +45,13 @@ const localTracks = {
 };
 
 
-const waitForPublished = (track, expectedPublished, description) => {
+const waitForPublished = (meeting, expectedPublished, description) => {
```

Comment:
It would be much better if this function had the stream as an argument and checked that not just any stream, but that the right stream was published or unpublished, Otherwise we might end up with flakey tests.